common/src/View/MapFrame.cpp

```diff
@@ -669,7 +655,6 @@ namespace TrenchBroom {
             connect(m_autosaveTimer, &QTimer::timeout, this, &MapFrame::triggerAutosave);
             connect(qApp, &QApplication::focusChanged, this, &MapFrame::focusChange);
             connect(m_gridChoice, QOverload<int>::of(&QComboBox::activated), this, [this](const int index) { setGridSize(index + Grid::MinSize); });
-            connect(QApplication::clipboard(), &QClipboard::dataChanged, this, &MapFrame::updatePasteActions);
```

Comment:
This needs to stay and call `updateActionState` instead of `updatePasteActions` I think, otherwise you won't be able to paste if you launch TB with an empty clipboard and then copy something.

Reply:
Right, this works now in b8dedff586e9864db9b3e541d9919a89cdd90cd9. I'm just triggering an update to all menu items.